local.bib

```diff
@@ -0,0 +1,7 @@
+@Misc{DMTN-140,
+  author = {G. Comoretto, L. P. Guy et all},
```

Comment:
I think the correct format uses "and" rather than comma.

Please add this to the master bib file rather than keeping it here.